### PR TITLE
Add competitor number. This fixes #379

### LINKED ISSUE #379: Add competitor number to FMC sheet
Add a line for competitor to write down his number of the competition. This will improve score-taking in big FMC competition like FMC Asia.

### PR DESCRIPTION
**Pull request by campos20**
We'll need to ask translators for the key Competition Number.

captura de tela de 2019-01-14 17-01-38

captura de tela de 2019-01-14 17-01-49

### CODE REVIEW

File: webscrambles/src/i18n/en.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ en:
         graded: Graded by
         result: Result
         competition: Competition
+        compNumber: Competitor Number
```
[the remark this answers is not in the document]
Comment (campos20) [reply webscrambles/src/i18n/en.yml]: All done for now, waiting for 688.

**Comment by Baiqiang** [comment]
Nice!

**Comment by campos20** [comment]
Just as a reference: it took 17 days for translators to [give ok](https://github.com/thewca/tnoodle/pull/388) here.

**Comment by suushiemaniac** [comment]
What's the status here?

**Comment by campos20** [comment]
I think we don't have a definite expression here, but this is taking too long. I think we could decide and proceed.

**Comment by campos20** [comment]
Btw, I'd go with Registrant Number.

**Comment by viroulep** [comment]
As mentioned in the issue's comment the WCIF is using "registrant id", the term used by the Regulations is ["competition registrant ID"](https://www.worldcubeassociation.org/regulations/guidelines.html#E2c1+), so I think we should use "Registrant ID".

**Comment by campos20** [comment]
Gone with Registrant ID. Merging after Travis says "yes".

**Comment by campos20** [comment]
Captura de tela de 2019-08-15 14-29-14

Captura de tela de 2019-08-15 14-28-59

**Comment by campos20** [comment]
I forgot. Travis doesn't like new translations.

**Pull request merged**